CONTRIBUTING.rst

```diff
@@ -105,20 +105,18 @@ encourage usage of `pipenv` and `pyenv`:
    (code style), PEP257 (documentation), flake8 as well as build the Sphinx
    documentation and run doctests.
 
-   Before you submit a pull request, please reformat the code using yapf_.
+   Before you submit a pull request, please reformat the code using black_.
 
    .. code-block:: console
 
-      $ yapf -irp .
+      $ black .
 
-   You may want to set up yapf_ styling as a pre-commit hook to do this
-   automatically:
-
-   .. code-block:: console
-
-      $ curl https://raw.githubusercontent.com/google/yapf/master/plugins/pre-commit.sh -o .git/hooks/pre-commit
-      $ chmod u+x .git/hooks/pre-commit
+   You may want to set up black_ styling as a pre-commit_ hook to do this
```

Comment:
We should add the `.pre-commit-config.yaml` mentioned in https://github.com/psf/black#version-control-integration to our repo to make this easier

Reply:
I agree - we already have it set up like this in the notebooks and gateway repos, would be nice to have consistency